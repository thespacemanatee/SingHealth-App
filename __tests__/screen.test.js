import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { configure } from 'enzyme';
configure({ adapter: new Adapter() });

describe('MyComponent', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<button debug />);
  
    expect(component).toMatchSnapshot();
  });
});