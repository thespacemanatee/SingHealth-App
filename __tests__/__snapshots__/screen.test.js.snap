// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
